pandas/core/reshape/melt.py

```diff
@@ -199,6 +200,10 @@ def wide_to_long(df, stubnames, i, j, sep="", suffix=r'\d+'):
 
         .. versionadded:: 0.20.0
 
+        When all suffixes are numeric, they are cast to int64/float64.
+
+        .. versionadded:: 0.22.0
```

Comment:
Can you use versionchanged instead of versionadded and format it this like:

```
.. versionchanged:: 0.22.0
    When all suffixes are numeric, they are cast to int64/float64.
```

(see http://www.sphinx-doc.org/en/stable/markup/para.html#directive-versionadded)

Reply:
Sure, I just pushed this change. It looks a little weird now though because there is a versionadded directly above it.

```
...
suffix : str, default '\\d+'
        A regular expression capturing the wanted suffixes. '\\d+' captures
        numeric suffixes. Suffixes with no numbers could be specified with the
        negated character class '\\D+'. You can also further disambiguate
        suffixes, for example, if your wide variables are of the form
        Aone, Btwo,.., and you have an unrelated column Arating, you can
        ignore the last one by specifying `suffix='(!?one|two)'`

        .. versionadded:: 0.20.0

        .. versionchanged:: 0.22.0
            When all suffixes are numeric, they are cast to int64/float64.
```